package.json

```diff
@@ -28,9 +28,11 @@
     "standard": "^6.0.6"
   },
   "dependencies": {
-    "electron-prebuilt": "^1.2.2",
+    "electron": "^1.4.1",
+    "electron-prebuilt": "^1.4.1",
```

Comment:
I just came across this post and seems that `electron-prebuild` is deprecated and maybe we can simply keep the `electron` one?